site/_data/releases.yaml

```diff
@@ -3,10 +3,10 @@ stable:
   tag: euwe-2
   filename: euwe-2
 
-# prerelease:
-#  name: Euwe-1-RC2
-#  tag: euwe-1-rc2
-#  filename: euwe-1-rc2
+prerelease:
+ name: Fine-1-Beta
+ tag: fine-1-beta
+ filename: fine-1-beta
```

Comment:
I notice in the builds that the tag/filename is `fine-1-beta1` instead of `fine-1-beta`